Website/index.js

```diff
@@ -2,18 +1,53 @@
+var baseURL = ["/sc_duty_list/api/", "http://localhost:3000/"][0];
+var online = (window.navigator.onLine ? true : false);
 
-$("#start").val(formatDate(new Date()));
+var dataData = {};
+var dataFlag;
+var dataNames;
+
+
+$.getJSON(`Data/data.json`, function (data) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8406024)
```suggestion
$.getJSON("Data/data.json", function (data) {
```